docs/explanation/tracks-and-upgrades.rst

```diff
@@ -0,0 +1,66 @@
+=================================
+MicroOVN snap tracks and upgrades
+=================================
+
+MicroOVN is distributed as a snap. As such, it utilises ``tracks`` to manage
+and control package upgrades. MicroOVN has dedicated tracks for supported LTS
+versions of OVN (e.g. ``22.03/stable``, ``24.03/stable``). These dedicated
+tracks should be used to install production deployments. They are guaranteed
+to always contain the same major version of OVN and therefore any automatic
+upgrades within the track won't cause incompatibilities across cluster members.
+
+Avoid using ``latest`` track for purposes other then development, testing or
+experimentation as it receives updates from the ``main`` development branch.
+It can contain experimental features and does not provide guarantees regarding
+compatibility of cluster members running different revisions from this track.
+
+
+Minor version upgrades
+----------------------
+
+Dedicated major version tracks of MicroOVN (e.g. ``24.03/stable``) will
+automatically receive minor version upgrades whenever the minor upgrade for
+the ``OVN`` package becomes available in the ``Ubuntu`` repository. They may
+also receive updates regarding MicroOVN itself in form of features or bugfixes
+if it's deemed that the backport is warranted.
+
+We try to keep the updates of dedicated stable tracks to minimum and any
+automatic upgrades within branch should not cause data plane outage.
```

Comment:
This section is about minor version upgrades that happen automatically. I thought that we added [pre-refresh hook](https://github.com/canonical/microovn/blob/ec59185100bbbb0417df20f90f44642590a07947/snap/hooks/pre-refresh#L7-L17) that does not clean up OVS datapaths to avoid DP outages during refresh.

Reply:
There is indeed a pre-refresh hook that aims to minimize the down time, but there will inevitably be some down time as the daemons restart and re-initializes the data path.

Reply:
I wonder if we should just document a recommendation for using `snap hold` in the production

Reply:
It's definitively worth highlighting the existence of the feature, but we need to be careful how we phrase it, we do not want users running into security risks on the back of something we recommended.